### PR TITLE
W-14416426 - RPA Manager release notes 2023-11/1.

### PR DESCRIPTION
**Pull request by jhammann72**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+[[november-02-2023]]
+== November 02, 2023
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Now global variables and credentials in the credentials pool do not have a limitation. | W-14321062
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| Global variables and credentials in the credentials pool no longer have a limitation. | W-14321062
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 2]
```diff
@@ -2,6 +2,31 @@
...
+| Processes now execute without interruption on RPA Bots. Under certain circumstances processes stopped executing. | W-14329929
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| Processes no longer stop executing on RPA Bots. | W-14329929
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 3]
```diff
@@ -2,6 +2,31 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[november-02-2023]]
+== November 02, 2023
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
[[november-3-2023]]
== November 3, 2023
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 4]
```diff
@@ -2,6 +2,31 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[november-3-2023]]
+== November 3, 2023
+
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc:7]: Is the "What's New" section missing? As you have fixed issues, you may say in such a section something like "This new version fixes issues and updates dependencies", or something similar.

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 5]
```diff
@@ -2,6 +2,31 @@
...
+| Processes no longer stop executing on RPA Bots. | W-14329929
+|===
+
+// == Known Issues
+include::partial$release-notes/rn-headings.adoc[tag=knownIssues,leveloffset=+2]
+
+[%header,cols="3a,3a,1a"]
+|===
+|Known Issue |Workaround | ID
+| Editing a process while you copy it causes RPA Builder to fail to import the copied process. | Before making any changes to the original process, import the copy in RPA Builder and then upload the copy to RPA Manager. After uploading the copy to RPA Manager, you can modify the original process and the copy without experiencing any issues. | W-12472442 / W-11954705
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc:24]:
```suggestion
| Editing a process while you copy it causes RPA Builder to fail to import the copied process. | Before making any changes to the original process, import the copy in RPA Builder and then upload the copy to RPA Manager. After uploading the copy to RPA Manager, you can modify the original process and its copy without experiencing any issues. | W-12472442 / W-11954705
```

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**